package.json

```diff
@@ -36,7 +36,10 @@
       "config": "commitizen.config.js"
     }
   },
-  "dependencies": {},
+  "dependencies": {
+    "enzyme-adapter-react-16": "^1.5.0",
```

Comment:
These should be `devDependencies`.

Reply:
Yeah, stupid mistake on my part.  Should I git commit --amend it or is it best to just push another commit?

Reply:
Another commit is fine